### ISSUE TITLE
Error after update from 2.0.4 to 2.1.0 in Safari

### ISSUE THREAD

**Issue by ddzobov** [bootstrap-vue/bootstrap-vue#4402]
```
[Vue warn]: Error in v-on handler: "TypeError: undefined is not an object (evaluating 'this.$refs.tbody.$el')"

found in

---> <BTr>
       <BTbody>
         <BTable>
           <BServerTable> at resources/js/components/BServerTable.vue
             <Table> at resources/js/pages/System/Table.vue
               <System> at resources/js/pages/System.vue
                 <LayoutComponent> at resources/js/LayoutComponent.vue
                   <AppComponent> at resources/js/App.vue
                     <Root>
```

Error appearing in Safari. In Chrome i can not reproduce it.

### Versions

**Libraries:**

- Vue 2.6.10
- BootstrapVue: 2.1.0

### Environment
- Mac OS Catalina 10.15.1
- Safari 13.0.3
- Chrome 78.0.3904.97

**Comment by tmorehouse** [comment]
Duplicate: #4384
Fixed by: #4388

Will be available in release v2.2.0

**Comment by tmorehouse** [comment]
BootstrapVue v2.2.0 has been released.
